src/smt/command.h

```diff
@@ -565,6 +565,28 @@ class CVC4_PUBLIC CheckSatCommand : public Command
   std::string getCommandName() const override;
 }; /* class CheckSatCommand */
 
+class CVC4_PUBLIC CheckSatAssumingCommand : public Command
```

Comment:
This class should be documented and conform to the coding standards (protected after public etc.).  See recent additions to this file (e.g. DefineFunctionRecCommand).